src/app/screens/LNURLChannel/index.tsx

```diff
@@ -115,6 +122,22 @@ function LNURLChannel() {
               heading={`${t("content_message.heading")}:`}
               content={uri}
             />
+            <div className="flex">
+              <Checkbox
+                id="is_private_channel"
+                name="private channel checkbox"
```

Comment:
Can we use a proper name here? (it's allowed and works, but to me this feels strange)

Reply:
```suggestion
                name="is_private_channel_checkbox"
```